### ISSUE TITLE
Training single track issues

### ISSUE THREAD

**Issue by Harry-Westwood** [Harry-Westwood/Y4-Project-InterNeuralStellar#37]
@grd349 
Hin and I have been training a lot of neural nets to try and get one which adequately fits to the data.
I have had some success using an architecture of 60 neurons per layer with 5 hidden layers, whose HR diagram is shown below:
![image](https://user-images.githubusercontent.com/56080854/75274508-de39ec80-57fa-11ea-9c92-0c37e1526e1e.png)
(I am currently increasing the regularization to smooth this out more, but it currently has good loss)
We tried using batch normalisation but the red bump kept appearing in the wrong place and try as we might to get rid of it by employing a combination of decreased architecture and regularization the results were fairly random: sometimes the bump would remain and other times it would disappear but would effect the shape of the rest of the track for the worse. 
With batch normalisation, different tracks were consistently bad but without batch normalisation an architecture that works for one track doesn't seem to work for all others. Hin will explain this further as he has been using the setups that I have found and trained on different tracks to me.
